samples/ask-akka-agent/src/main/java/akka/ask/Bootstrap.java

```diff
@@ -14,18 +14,20 @@
 @Setup
 public class Bootstrap implements ServiceSetup {
 
+  private Config config;
   // end::knowledge[]
 
   public Bootstrap(Config config) {
+    this.config = config;
+    // end::mongodb[]
     KeyUtils.checkKeys(config);
+    // tag::mongodb[]
```

Comment:
The reason for this is that currently we dont show anywhere in the guide the contents of `KeyUtils`.. so instead of fixing it by having another step where the use needs to create it, I think it's fine to just not use it on the code the user needs to past.